app/controllers/api/braintree_controller.rb

```diff
@@ -24,7 +24,16 @@ def manage_transaction(params)
     Payment.write_transaction(page: page, transaction: result)
 
     if result.success?
-      ManageAction.create( params[:user].merge(page_id: params[:page_id]) )
+      begin
+        ManageBraintreeDonation.create(params: params, braintree_result: result )
+      rescue
+        # There's a possibility that trying to create the donation on the queue is going to
+        # error in some way, like missing member data. If that happens, we shouldn't fail on the user
+        # but we should log the error to let ourselves know that something went wrong and we need to import
+        # the data correctly.
+
+        ## For comments: How should we log this?
```

Comment:
Perhaps move `Payment.write_successful_transaction(action: action, transaction_response: result)` into the rescue block. I'm not sure what might raise here. If it's our own code raising, I'd rather it failed fast so we can fix the broken code.